ironfish/src/rpc/routes/multisig/createTrustedDealerKeyPackage.ts

```diff
@@ -0,0 +1,85 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { generateKey, splitSecret } from '@ironfish/rust-nodejs'
+import * as yup from 'yup'
+import { ApiNamespace } from '../namespaces'
+import { routes } from '../router'
+
+export type CreateTrustedDealerKeyPackageRequest = {
+  minSigners: number
+  maxSigners: number
+  participants: Array<{
+    identifier: string
```

Comment:
I was thinking about whether we'd want this to be `Identifier` or `Identity`

I think that since the response uses `Identifier` it probably makes more sense to use `Identifier` as the input rather than forcing the wallet or RPC to transform between the two

Reply:
Hmm, it is a good point. If we imagine the case where the identities are provided to the trusted dealer before key generation, they will have the ids in identity format, which will need to be stored. At some point we will have to do the transform from identity to identifier though